modules/mono/glue/GodotSharp/GodotSharpEditor/Compat.cs

```diff
@@ -7,6 +7,8 @@
 
 namespace Godot;
 
+#pragma warning disable IDE0040 // Add accessibility modifiers. This is currently disabled because of https://github.com/dotnet/roslyn/issues/52904
```

Comment:
```suggestion
#pragma warning disable IDE0040 // Add accessibility modifiers.
    // TODO: This is currently disabled because of https://github.com/dotnet/roslyn/issues/52904
```